pkg/model/hornet/solid_entry_points.go

```diff
@@ -36,6 +36,11 @@ func (s *SolidEntryPoints) Contains(txHash Hash) bool {
 	return exists
 }
 
+func (s *SolidEntryPoints) Index(txHash Hash) (milestone.Index, bool) {
+	index, exists := s.entryPointsMap[string(txHash)]
+	return index, exists
```

Comment:
really?

Reply:
![grafik](https://user-images.githubusercontent.com/32371094/84840163-12058000-b03f-11ea-955c-2522877100c9.png)
yes!

Reply:
liked that german IDE

Whow, go has surprising behavior though